### PR TITLE
Choon Wei's PPP draft

### PR DESCRIPTION
**Pull request by chwoozy**

### CODE REVIEW

File: docs/team/yungweezy.md [hunk 1]
```diff
@@ -3,10 +3,60 @@ layout: page
...
-* **Code contributed**: [RepoSense link]()
+Given below is a summary of my contributions to the development of TutorBuddy.
+
+### Major Enhancement
+
+* Overlapping Sessions Check
+    - **What it does**: This feature helps to validate the specified session date in the `add_session` and `add_rec_session` command.
+    - **Justification**: This feature provides an important check that informs the tutor if he/she already has a session that falls within the specified date.
+      This prevents the tutor from scheduling two students on the same date and time, reducing the possibilities of awkward session cancellation/postponement.
+    - **Rationale**: The intricacies of this feature was very much underestimated. Everytime `add_session` and `add_rec_session` is called, the check will take place.
```
Comment (enhao25) [inline docs/team/yungweezy.md]: Hmmmm is "Rationale" supposed to be "Highlight"?

**Review by enhao25 — APPROVED** [review]
LGTM! Only have 1 comment which I think you could have missed out. Else all is good and thanks for the effort in the project 👍🏻 💯

**Review by nowknowing — APPROVED** [review]
LGTM

Just that Rationale and Justification might mean the same thing. 
Possibly what’s under rationale cld be names “Details” / “Implementation” etc

File: docs/team/yungweezy.md (hunk 1 above)
Comment (nowknowing) [reply docs/team/yungweezy.md]: Ah yes i agree probably highlight

Comment (chwoozy) [reply docs/team/yungweezy.md]: Thank you good sir and mdm, I will make the necessary changes :)

**Pull request merged**